source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -849,7 +866,12 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
        * The service id for which the pool was cleared for in load balancing mode.
        * See load balancer specification for more information about this field.
        */
-      serviceId: Optional<ObjectId>
+      serviceId: Optional<ObjectId>;
+
+      /**
+       * A flag whether the pool forced closing "in use" connections as part of the clear.
+      */
+      closeInUseConnections: Optional<Boolean>;
```

Comment:
cc @BorisDog 

@DmitryLukyanov and I discussed the naming of this option and how it can lead to confusion in driver implementations, namely that driver authors may try to implement this by "closing" (in the CMAP sense, thereby emitting ConnectionClosedEvents) in use connections directly from the background thread. To keep consistent with the existing Connection lifecycle model, it would instead be better if these connections were marked as perished (e.g. by closing the underlying TCP socket), and then the "closing" would happen in checkIn as is the case for other kinds of network errors currently.

In order to better communicate this requirement, we proposed renaming this option to `interruptInUseConnections` and to add a definition of "interrupt", which would read something like "cancels whatever task the connection is currently performing and marks the connection as perished (e.g. by closing the underlying socket)".

Does this sound good to you?

Reply:
done